crates/db_schema/src/impls/comment.rs

```diff
@@ -116,6 +117,11 @@ impl Comment {
       None
     }
   }
+
+  pub fn local_url(&self, settings: &Settings) -> LemmyResult<DbUrl> {
+    let domain = settings.get_protocol_and_hostname();
```

Comment:
protocol_and_hostname is the only var used, so it seems excessive to pass around the entire settings object for all of these. Maybe if it used a lot things from settings, but its only using one `&str`.

Reply:
Its only a reference so it makes no difference for performance. Otherwise it would be extremely verbose to call, and there would be no type safety with a plain string.